src/ctia/features_service.clj

```diff
@@ -24,6 +24,7 @@
 (tk/defservice features-service
   FeaturesService
   [[:ConfigService get-in-config]]
+  ;;TODO build a proper init to prepare fields only once
```

Comment:
@agzam thoughts?

Reply:
Just to avoid reading it from the config every time? What's the bottleneck here?